src/components/About.tsx

```diff
@@ -29,13 +29,13 @@ const About = () => {
               onPress={() => setModalVisible(!modalVisible)}>
               <Image
                 style={styles.imgStyle}
-                source={require('../img/close.png')}
+                source={require(BTN_CLOSE)}
               />
             </Pressable>
 
             <Text>
               Created By: {'\n'}
-              <Image source={require('../img/khalislogo150.png')} />
+              <Image source={require(MAIN_LOGO)} />
```

Comment:
Just curious, if `require` is required here.. passing just the path of assets, not works?

Reply:
without require will work only for the remote images uri, local images need require to map it correctly on compile time on diffrent devices folder structure